core/templates/dev/head/components/sharing_links_directive.html

```diff
@@ -2,19 +2,19 @@
   <ul class="oppia-sharing-links" layout="<[layoutType]>" layout-align="<[layoutAlignType]>">
     <li>
       <a ng-href="https://plus.google.com/share?url=<[serverName]>/explore/<[explorationId]>" onclick="return !window.open(this.href, '', 'height=600, width=600, menubar=no, toolbar=no, resizable=yes, scrollbars=yes')" ng-click="registerShareExplorationEvent('gplus')" target="_window">
-        <img class="share-option-img" src="/images/general/gplus.png" alt="Google+">
+        <img class="share-option-img" src="/static/{{cache_slug}}/images/general/gplus.png" alt="Google+">
```

Comment:
No. Define a variable with the URL in the JS, and refer to that. E.g.

ng-src="gplusUrl"

On Sunday, July 3, 2016, Vishal Gupta <notifications@github.com> wrote:

> In core/templates/dev/head/components/sharing_links_directive.html
> <https://github.com/oppia/oppia/pull/2208#discussion_r69389099>:
>
> > @@ -2,19 +2,19 @@
> >    <ul class="oppia-sharing-links" layout="<[layoutType]>" layout-align="<[layoutAlignType]>">
> >      <li>
> >        <a ng-href="https://plus.google.com/share?url=<[serverName]>/explore/<[explorationId]>" onclick="return !window.open(this.href, '', 'height=600, width=600, menubar=no, toolbar=no, resizable=yes, scrollbars=yes')" ng-click="registerShareExplorationEvent('gplus')" target="_window">
> > -        <img class="share-option-img" src="/images/general/gplus.png" alt="Google+">
> > +        <img class="share-option-img" src="/static/{{cache_slug}}/images/general/gplus.png" alt="Google+">
>
> You mean, use something like:
> ng-src='/static/{{GLOBALS.CACHE_SLUG}}/images?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/oppia/oppia/pull/2208/files/126df3730393afd91f660eb4f210060541710fee#r69389099>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKFeyqySIzgm-WDKoZbEUrICVvc-zYsjks5qR5AqgaJpZM4JDv96>
> .
>